src/pages/oauth-error-view/oauth-error-view.js

```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom'
+import './oauth-error-view.scss'
+import qs from 'qs'
+import routes from "../../routes.js"
+
+const getError = search => {
+  return qs.parse(search, { ignoreQueryPrefix: true }).error_description
```

Comment:
Is `error_description` a parameter that auth0 will provide when redirecting to this page? 
Also, does all of the config for going to this page exist on the auth0 server side?  No config that needs to happen here in the front end?

Reply:
Yes - auth0 populates it. 
We will need to modify the auth0 config - going to do that in auth0-setup next. There is no config in the front end needed.